### PR TITLE
Fix/contribution bugs

### PR DESCRIPTION
**Pull request by haydencleary**

### CODE REVIEW

File: src/pages/ProjectDetails/Rewards/RewardForm/WorkItemSidePanel/EmptyState/index.tsx [hunk 1]
```diff
@@ -1,28 +1,17 @@
...
         <img src={emptyStateLogo}></img>
       </div>
       <div className="text-center font-belwe text-2xl font-normal text-greyscale-50">
         {T("reward.form.contributions.emptyState.title")}
       </div>
-      <div className="text-center font-walsheim text-base font-normal text-greyscale-50">
-        {T("reward.form.contributions.emptyState.subtitle", {
-          time: getFormattedTimeUS(isoDate),
-        })}
-      </div>
```
Comment (kaelsky) [inline src/pages/ProjectDetails/Rewards/RewardForm/WorkItemSidePanel/EmptyState/index.tsx:25]: This is the `WorkItemSidePanel` EmptyState, I believe we should display indexation time here

Comment (haydencleary) [reply src/pages/ProjectDetails/Rewards/RewardForm/WorkItemSidePanel/EmptyState/index.tsx:25]: @kaelsky I checked with Gregoire already. The reason is soon we should have real-time data so the message about the sync time will be obsolete. I changed the message for the contributions table and he wanted me to update this one too.

**Review by kaelsky — CHANGES_REQUESTED** [review]
Some side effect in chromatics and stories than need a mocked provider 

[Contribution](https://www.chromatic.com/component?appId=63984b27c30bb6c931960c17&csfId=contribution&buildNumber=3789):[Has Reward](https://www.chromatic.com/component?appId=63984b27c30bb6c931960c17&csfId=contribution&specName=Has%20Reward&buildNumber=3789)

File: src/components/Contribution/ContributionDate.tsx [hunk 2]
```diff
@@ -62,8 +62,8 @@ export function ContributionDate({
 
   return (
     <>
-      <Tooltip id={tooltipId} clickable {...rest}>
-        <div className={cn("flex items-center gap-2 px-1 py-2", className)}>
+      <Tooltip id={tooltipId} clickable {...rest} position={TooltipPosition.TopStart}>
+        <div className={cn("flex items-center gap-2 px-1 py-2 text-sm", className)}>
```
Comment (kaelsky) [inline src/components/Contribution/ContributionDate.tsx:66]: This seems to affect chromatics change in WorkItems tooltip

**Review by kaelsky — APPROVED** [review]
LGTM ! 💪

**Pull request merged**